stock_quant_sale_order/i18n/ja.po

```diff
@@ -0,0 +1,116 @@
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_quant_sale_order
+#: model:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
+msgid "Cancel"
+msgstr "取消"
+
+#. module: stock_quant_sale_order
+#: model:ir.actions.act_window,name:stock_quant_sale_order.action_quant_sale_order
+#: model:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
+msgid "Create Sale Order"
+msgstr "販売オーダ作成"
+
+#. module: stock_quant_sale_order
+#: model:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
+msgid "Create sale order for selected quants."
+msgstr "選択した保管ロットの販売オーダを作成します。"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_create_uid
+msgid "Created by"
+msgstr "作成者"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_create_date
+msgid "Created on"
+msgstr "作成日"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_partner_id
+msgid "Customer"
+msgstr "顧客"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_display_name
+msgid "Display Name"
+msgstr "表示名"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_id
+msgid "ID"
+msgstr "ID"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard___last_update
+msgid "Last Modified on"
+msgstr "最終更新日"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_write_uid
+msgid "Last Updated by"
+msgstr "最終更新者"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_write_date
+msgid "Last Updated on"
+msgstr "最終更新日"
+
+#. module: stock_quant_sale_order
+#: code:addons/stock_quant_sale_order/wizard/stock_quant_sale_order.py:36
+#, python-format
+msgid "Please select quants that are in the same Location."
+msgstr "同じロケーション内の保管ロットを選択してください。"
+
+#. module: stock_quant_sale_order
+#: model:ir.ui.view,arch_db:stock_quant_sale_order.quant_sale_order_wizard_form
+msgid "Quant Transfer"
+msgstr "内部移動"
+
+#. module: stock_quant_sale_order
+#: model:ir.model.fields,field_description:stock_quant_sale_order.field_quant_sale_order_wizard_team_id
+msgid "Sales Channel"
+msgstr "販売チャネル"
+
+#. module: stock_quant_sale_order
+#: code:addons/stock_quant_sale_order/wizard/stock_quant_sale_order.py:49
+#, python-format
+msgid "Some of the selected quants are already create sale order:%s"
+msgstr "選択した保管ロットはすでに他の販売オーダに属しています。\n以下の販売オーダを処理してください：%s"
```

Comment:
@yostashiro Can you review this translation?

Reply:
See the comment in .py file.